tests/etcd_length_test/ip_list/ips_list.txt

```diff
@@ -0,0 +1,9 @@
+cluster
```

Comment:
I'd remove this file from PD, it contains internal IPs

Reply:
this file removed from the PR.